HintList.py

```diff
@@ -901,7 +901,7 @@ def tokens_required_by_settings(world: World) -> int:
     'Shadow Temple After Wind Enemy Chest':                        ("#mummies guarding a ferry# hide", None, 'exclude'),
     'Shadow Temple After Wind Hidden Chest':                       ("#mummies guarding a ferry# hide", None, 'exclude'),
     'Shadow Temple Spike Walls Left Chest':                        ("#walls consumed by a ball of fire# reveal", None, 'exclude'),
-    'Shadow Temple Invisible Floormaster Chest':                   ("shadows in an #invisible maze# guard", None, 'exclude'),
+    'Shadow Temple Invisible Floormaster Chest':                   ("a #Floormaster in the house of the dead# guards", "a #Floormaster in the Shadow Temple# guards", 'exclude'),
```

Comment:
I'd change to "**the** floormaster" since there's only one

Reply:
Wow there really is only one. Somehow I never even realized that. And none in MQ.

Will change that later when i have time, thanks.